suit_generator/cmd_cache_create.py

```diff
@@ -13,23 +13,37 @@
 
 CACHE_CREATE_CMD = "cache_create"
 
+
 def add_arguments(parser):
     """Add additional arguments to the passed parser."""
     cmd_cache_create_arg_parser = parser.add_parser(CACHE_CREATE_CMD, help="Create raw cache structure.")
 
     cmd_cache_create_arg_parser.add_argument(
-        "--input", required=True, action="append",
+        "--input",
+        required=True,
+        action="append",
         help="Input binary with corresponding URI, passed in format <URI>,<INPUT_FILE_PATH>."
-        + "Multiple inputs can be passed."
+        + "Multiple inputs can be passed.",
     )
     cmd_cache_create_arg_parser.add_argument("--output-file", required=True, help="Output raw SUIT DFU cache file.")
-    cmd_cache_create_arg_parser.add_argument("--eb-size", type=int, help="Erase block size in bytes (used for padding).",
-                                             default=16)
+    cmd_cache_create_arg_parser.add_argument(
+        "--eb-size", type=int, help="Erase block size in bytes (used for padding).", default=16
+    )
 
 
 def add_padding(data: bytes, eb_size: int) -> bytes:
-    data_len = len(data)
-
+    """
+    Add padding to the given data to align it to the specified erase block size.
+
+    This function ensures that the data is padded to a size that is a multiple of the erase block size (eb_size).
+    The padding is done according to the CBOR (Concise Binary Object Representation) format.
```

Comment:
You may also add a more detailed explanation here as this function does not append a dumb, constant padding:

The padding is done by appending a CBOR  key-value pair with empty URI as the key and byte-string-encoded zeros as the value.